src/cookbook/testing/unit/mocking.md

```diff
@@ -54,14 +54,10 @@ so define that dependency in the `dependencies` section.
 To run the required code generation, add the `build_runner` dependency
 in the `dev_dependencies` section.
 
-```yaml
-dependencies:
-  http: <newest_version>
-dev_dependencies:
-  flutter_test:
-    sdk: flutter
-  mockito: <newest_version>
-  build_runner: <newest_version>
+To add the packages, run `flutter pub add`:
```

Comment:
```suggestion
To add the dependency, run `flutter pub add`:
```

Reply:
Note that here it is referring to multiple dependencies (mockito, build_runner, etc.) and not a single one

Reply:
Oops, sorry. How about:

```suggestion
To add the dependencies, run `flutter pub add`:
```